features/step_definitions/json_api_average.rb

```diff
@@ -9,9 +9,9 @@
 end
 
 Given(/^a set of ratings for several songs$/) do
-  6.times { |i| create :song_rating, value: i + 1, ratable_id: 1 }
-  6.times { |i| create :song_rating, value: i + 1, ratable_id: 2 }
-  6.times { |i| create :song_rating, value: i + 1, ratable_id: 3 }
+  6.times { |i| create :song_rating, value: (i + 1) - 1, ratable_id: 1 }
+  6.times { |i| create :song_rating, value: (i + 1) - 1, ratable_id: 2 }
+  6.times { |i| create :song_rating, value: (i + 1) - 1, ratable_id: 3 }
```

Comment:
It's not pretty, but it was the quickest way to get the values to increment between 0 and 5 which is the new limit set on that column. Otherwise the values start at 1 and end at 6, which throws an error.